.github/workflows/update-gradle-wrapper.yml

```diff
@@ -10,5 +10,7 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
+        with:
+          persist-credentials: false
```

Comment:
I don't think this will work

Reply:
This actually does work and has been tested before it was ever submitted, and it was caught by the `zizmor` rust library.